content/docs/user-guide/plots.md

```diff
@@ -0,0 +1,420 @@
+`x` axis we will have `actual_class` field. Note that DVC will assume that
+`actual_class` is inside `confusion_matrix_data.csv`.
+
+We can provide multiple columns/fields from the same file:
+
+```yaml
+#dvc.yaml
+---
+plots:
+  multiple_series:
+    y:
+      logs.csv: [accuracy, loss]
+    x: epoch
+```
+
+In this case, we will take `accuracy` and `loss` fields and display them agains
+`epoch` column, all coming from `logs.csv` file.
+
+We can source the data from multiple files too:
+
+```yaml
+#dvc.yaml
+---
+plots:
+  multiple_files:
+    y:
+      train_logs.csv: accuracy
+      test_logs.csv: accuracy
+    x: epoch
+```
+
+In this case we will plot `accuracy` field from both `train_logs.csv` and
+`test_logs.csv` against the `epoch`. Note that both files have to have `epoch`
+field.
+
+### Available configuration fields
+
+- `x` - field name from which the X axis data comes from. An auto-generated
+  _step_ field is used by default. It has to be a string.
+
+- `y` - field name from which the Y axis data comes from.
+  - Top-level plots: It can be a string, list or dictionary. If its a string or
+    list, it is assumed that plot ID will be the path to the data source.
+    String, or list elements will be the names of data columns or fields withing
+    the source file. If this field is a dictionary, it is assumed that its keys
+    are paths to data sources. The values have to be either strings or lists,
+    and are treated as column(s)/field(s) within respective files.
+  - Plot outputs: It is a field name from which the Y axis data comes from.
+- `x_label` - X axis label. The X field name is the default.
+- `y_label` - Y axis label. If all provided Y entries have the same field name,
+  this name will be the default, `y` string otherwise.
+- `title` - Plot title. Defaults:
+  - Top-level plots: `path/to/dvc.yaml::plot_id`
+  - Plot outputs: Path to the file.
+
+Refer to the [`show` command] documentation for examples.
+
+[`show` command]: /doc/command-reference/plots/show#example-top-level-plots
+
+## Plot templates (data-series only)
+
+DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON specifications to
+create plots from user data. A set of built-in _plot templates_ are included.
+
+The `linear` template is the default. It can be changed with the `--template`
+(`-t`) option of `dvc plots show` and `dvc plots diff`. The argument provided to
+`--template` can be a (built-in) template name or a path to a [custom template].
+
+<admon type="tip">
+
+For templates stored in `.dvc/plots` (default location for custom templates),
+the path and the json extension are not required: you can specify only the base
+name, e.g. `--template scatter`.
+
+</admon>
+
+DVC has the following built-in plot templates:
+
+- `linear` - basic linear plot including cursor interactivity (default)
+- `simple` - simplest linear template (not interactive); Good base to create
+  [custom templates].
```

Comment:
To complement https://github.com/iterative/dvc.org/pull/3830#pullrequestreview-1057430160
```suggestion
  custom templates.
```